### DISCUSSION TITLE
Mermaid links don't work in markdown

### DISCUSSION THREAD

**Discussion by aiven-amartin** [community/community#17545] [Other Feature Feedback, Questions, & Ideas]
Raised https://github.com/github/docs/issues/17007, but was redirected here. Example below:

This mermaid doc should have links, but it doesn't 
```mermaid
sequenceDiagram
    autonumber
    A ->> B: Hello
    links A: {"google": "https://google.com", "github", "https://github.com"}
```
Working example: https://mermaid-js.github.io/mermaid-live-editor/edit#pako:eNpdj7sOwjAMRX8l8lzonqFSEUjsDCxe0sRtI_KA1BlQ1X-nJWXBm4-P7OsZdDQEEiZ6ZQqazlYNSXkMYi2VOYbsO0qlb8WhacRJiis5F8U9JmfKxNnwmEQrxYwwxDg4QpACYWR-TrKuCzvq6BGqlQ-Wx9z9OV9WnAUDVOApeWXNGm_eziDwSH7fbKhX2TEChmVV89MopouxHBPIXrmJKtjy395Bg-SU6SftL-7W8gEFqFaT

**Comment by rsheeter** [comment]
Another example:

```mermaid
flowchart TD
    subgraph A
        B
    end

    click B "https://github.com/" _blank
```

This does produce a node that is a link (though without a target, presumably due to https://github.com/mermaid-js/mermaid/issues/3077) but if you click them you an error page stating "This content is blocked. Contact the site owner to fix the issue."

Console logs _Refused to frame 'https://github.com/' because it violates the following Content Security Policy directive: "frame-src viewscreen.githubusercontent.com notebooks.githubusercontent.com"._

It would be very helpful to have a functional way to make graphs with links.
